packages/patternfly-4/react-docs/package.json

```diff
@@ -37,8 +37,8 @@
     "gatsby"
   ],
   "scripts": {
-    "docbuild": "node build/copyDocs.js && node --max_old_space_size=4096 ./node_modules/.bin/gatsby build",
-    "pr-build": "node build/copyDocs.js && node --max_old_space_size=4096 ./node_modules/.bin/gatsby build --prefix-paths",
```

Comment:
these were added to increase the available memory for the node process as gatsby is sometimes hitting it and then erroring out. Is this not needed anymore?

Reply:
The module resolution is the bigger problem. If Gatsby gets hoisted, it won't be in that path.

Reply:
Can we adjust the path to go to the relative root?

Reply:
We could, but could you convince me of when Gatsby could run out of memory? I haven't experienced it on Travis or CircleCI, which have 7.5GB and 4GB of RAM respectively.

Reply:
I'm more concerned about fixing this rare Gatsby issue with CircleCI, which happened [with](https://circleci.com/gh/patternfly/patternfly-react/35) and [without](https://circleci.com/gh/patternfly/patternfly-react/49) the --max_old_space flag. Maybe it is related to RAM... any ideas?

Reply:
Gatsby definitely uses a lot of ram, but if it runs out it would typically error out like this:
```
success createPages — 16.948 s
success createPagesStatefully — 0.120 s
success onPreExtractQueries — 0.014 s
success update schema — 6.081 s
success extract queries from components — 0.265 s
⠁
<--- Last few GCs --->

[11566:0x37ab5d0]    73868 ms: Mark-sweep 1416.3 (1495.9) -> 1416.3 (1479.9) MB, 915.0 / 0.0 ms  (+ 0.0 ms in 0 steps since start of marking, biggest step 0.0 ms, walltime since start of marking 915 ms) last resort
[11566:0x37ab5d0]    74780 ms: Mark-sweep 1416.3 (1479.9) -> 1416.3 (1479.9) MB, 912.6 / 0.0 ms  last resort


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x196898628799 <JSObject>
    0: builtin exit frame: captureStackTrace(aka captureStackTrace)(this=0x393b29c02311 <undefined>,0x2f691ed37cd1 <JSFunction CapturedTrace (sfi = 0x25975f15c491)>,0x328575c09329 <CapturedTrace map = 0x38644cc40001>)

    1: _then [/mnt/c/Users/brand/code/admitbrain/node_modules/bluebird/js/release/promise.js:~219] [pc=0x2c92629e6e1f](this=0x3470fb292b09 <Promise map = 0x38644cc400b1>,didFulfil...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
```
I have not seen the terser error before though, not too sure where that's coming from

Reply:
I found an environment variable `env NODE_OPTIONS=--max_old_space_size=4096`, that way a relative path isn't necessary.